src/tracks/playabletrack/wavetrack/ui/SpectrumView.h

```diff
@@ -65,12 +65,16 @@ class SpectralData{
 
    // The double time points is quantized into long long
    void addTimeFreqData(long long ll_sc, wxInt64 freq){
+      // Update the start and end sampleCount of current selection
       if(ll_sc > mEndT)
          mEndT = ll_sc;
-
       if(ll_sc < mStartT)
          mStartT = ll_sc;
 
+      // Using int division to round the sampleCount to the hop size of FFT to save computation
+      int hopSize = 128;
+      ll_sc = ll_sc / hopSize * hopSize;
+
```

Comment:
Question:  should this rounding happen before or after the update of mStartT and mEndT?